### PR TITLE
Add a msgpack content viewer.

### PR DESCRIPTION
**Pull request by tasn**
#### Description

This is a msgpack content viewer.
Is this of interest? If so, what kind of tests should I add if at all?

Thanks.

#### Checklist

 - [X] I have updated tests where applicable.
 - [X] I have added an entry to the CHANGELOG.

### CODE REVIEW

**Comment by Kriechi** [comment]
Yes - I think this would be a nice feature to have!

You can find similar tests for other contentviews under https://github.com/mitmproxy/mitmproxy/blob/master/test/mitmproxy/contentviews/test_json.py and related files in this folder.

File: setup.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
         "hyperframe>=5.1.0,<6",
         "kaitaistruct>=0.7,<0.9",
         "ldap3>=2.6.1,<2.8",
+        "msgpack>=1.0.0",
```
Comment (Kriechi) [inline setup.py:74]: Please add a `<2` upper bound, assuming they follow semver?

**Comment by tasn** [comment]
I can't run the tests locally for some reason. I'm getting:
```
pkg_resources.DistributionNotFound: The 'msgpack>=1.0.0' distribution was not found and is required by mitmproxy
ERROR: InvocationError for command /home/tom/projects/tmp/mitmproxy/.tox/py/bin/mitmdump --version (exited with code 1)
```

So I have no idea why the rest of the tests fail. If you have any idea, or you could maybe fix it, that would be great. I tried running mypy but already saw errors in JSON, and the errors it's complaining about in msgpack are not really errors. Maybe just missing typings?

File: setup.py (hunk 1 above)
Comment (tasn) [reply setup.py]: Done.

**Comment by Kriechi** [comment]
code coverage is complaining: https://github.com/mitmproxy/mitmproxy/pull/4096/checks?check_run_id=898723405#step:5:58

Otherwise, I think this looks good.
Could you maybe post a screenshot of an example here?

**Comment by tasn** [comment]
Fixed.

**Comment by mhils** [comment]
Awesome, thanks! 🍰

**Comment by tasn** [comment]
Great, thanks!

**Pull request merged**